src/main/java/com/bloomtech/asylumrgbea/Seeder.java

```diff
@@ -22,7 +26,35 @@ public class Seeder implements CommandLineRunner {
      */
     @Override
     public void run(String... args) throws Exception {
-
+//  --------------------------------------------------------------------------------------------------------------------
+//  |                                           Uncomment / Comment to write from csv to DB                             |
+//  --------------------------------------------------------------------------------------------------------------------
+//        String line = "";
+//        String splitBy = ",";
+//        List<AsylumCase> cases = new ArrayList<>();
+//        try
+//        {
+//            // csv is created using configurations/asylumCaseXlsxToCleanCsv.py
+//            BufferedReader br = new BufferedReader(new FileReader("C:path/to/csv"));
+//            int i = 0;
+//            br.readLine();
+//            while ((line = br.readLine()) != null)
+//            {
+//                String[] caseLine = line.split(splitBy);
+//                cases.add(new AsylumCase(caseLine[0],caseLine[2], caseLine[3],caseLine[4], caseLine[5], caseLine[1]));
+//                i++;
+//            }
+//            br.close();
+//        }
+//        catch (IOException e)
+//        {
+//            e.printStackTrace();
+//        }
+//        asylumCaseRepository.saveAll(cases);
+
+//  --------------------------------------------------------------------------------------------------------------------
```

Comment:
Is this the commented-out code you were talking about? I think this should be fine. Maybe we could it move it to another file so that file can only be run when it is necessary to run the file?

Reply:
Yes, it is and I really like that idea!